spinta/manifests/xsd2/helpers.py

```diff
@@ -338,7 +338,22 @@ def _post_process_resource_model(self):
             self.models.append(self.resource_model)
 
     def _post_process_refs(self):
-        pass
+        """
+        Links properties in all models to their target models based on xsd_ref_to and xsd_type_to.
+        """
+        for model in self.models:
+            for prop in model.properties.values():
+                if prop.xsd_ref_to:
+                    target_model: XSDModel = self.top_level_element_models.get(prop.xsd_ref_to) or self.top_level_complex_type_models.get(prop.xsd_ref_to)
```

Comment:
In this case it should look only in `top_level_element_models`. If it doesn't find the model there, it means that there's some bug in adding models to those lists.